src/skin/Theme.js

```diff
@@ -65,18 +65,17 @@ Theme.prototype.setSkin = function(comp, id, skin) {
 /**
  * Set up the asset loader and load files
  *
- * @method loadImage
+ * @method addImage
  * @param jsonPath {Array}
  */
-Theme.prototype.loadImage = function(jsonPath) {
+Theme.prototype.addImage = function(jsonPath) {
```

Comment:
what is the reason for renaming the function?

I would like to avoid unnecessarily breaking the API.